tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -261,7 +321,8 @@ class CompletionArgSuite extends BaseCompletionSuite {
         |}
         |""".stripMargin,
     "foo(rele@@)",
-    "foo(relevant)"
+    "foo(relevant)",
+    assumeCond = Some(!isScala3)
```

Comment:
This should work currently normally.

Reply:
Unfortunately, no it doesn't work in scala3.
Because `relevant` is searched from the scope, but scala3 doesn't support it yet.